source/internals/options.js

```diff
@@ -54,14 +54,15 @@ const askQuestion = (question, callback) => {
 };
 
 const checkAnswer = (answer, questionFunction, resolve) => {
-  if (answer === 'Y') {
+  answer = answer.toLowercase();
+  if (answer === 'y') {
```

Comment:
I think it's worth having both options in order to not break legacy.

Reply:
It will work on both because I am lowering the case

Reply:
Absolutely right!